doc/js_tutorials/js_assets/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "image_classification",
+  "version": "0.0.1",
+  "description": "An Electon.js example of image_classification using webnn-native",
```

Comment:
It makes sense to move Electron-specific stuff into sub-directory to avoid confusion.